.github/workflows/publish.yml

```diff
@@ -0,0 +1,32 @@
+# This workflow will publish an es-node release with a pre-build executable
+
+name: GitHub Actions ES Publish
+run-name: ${{ github.actor }} is publishing a release 🚀
+on:
+  push:
+    tags: 
+      - 'v*'
+jobs:
+  build:
+    runs-on: ubuntu-latest
```

Comment:
If github also support darwin and windows, we may also want to support it. Also is it x86 64-bit build? Some people   may run es-node in their MBP with ARM64, so it will be nice to support it too.

Reply:
now we support:

- es-node.v0.1.2.darwin-amd64 
- es-node.v0.1.2.darwin-arm64
- es-node.v0.1.2.linux-amd64 
- es-node.v0.1.2.windows-amd64